platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -365,7 +386,7 @@ protected boolean doLogin() throws LoginException {
                         }
                     }
                 } catch (Exception e) {
-                    boolean result;
+                    LOGGER.debug("Exception while getting roles for user.", e);
```

Comment:
❓ Is this debug message as necessary since you're throwing a checked exception on the next line that will presumably be logged elsewhere?